schema/snapcraft.yaml

```diff
@@ -190,6 +190,10 @@ properties:
                 - type: string
                   minLength: 1
                 - type: number
+          no-wrapper:
```

Comment:
Suggestion: make this "wrappers" enum that has possible values "legacy" and "none" as we may be able to provide some kind of useful wrappers later on.